cg/meta/orders/api.py

```diff
@@ -72,6 +72,13 @@ def submit(self, project: OrderType, order_in: OrderIn, user_name: str, user_mai
             ticket_number = self.ticket_handler.create_ticket(
                 order=order_in, user_name=user_name, user_mail=user_mail, project=project
             )
-
+        elif ticket_number != self.ticket_handler.TESTING_TICKET:
```

Comment:
I think we should handle this in another way. The fact that we are testing today with "123456" doesn't mean it's a invalid ticket number. We should change the way we test according to how the production works instead of changing how the production code works to how we test 😃 
```suggestion
```

Reply:
Implemented new subclass to Osticket that is instantiated when environment is stage

Reply:
After talking with @Mropat we agreed to remove the class and only use an the OSTICKET_EMAIL env variable